### ISSUE TITLE
Testing 3.2.0b2

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#82]
In GitLab by @kserradell on Jan 8, 2016, 09:11

When running a03i experiment, I get this error.

```python
Checking configuration files...
autosubmit_a03i.conf OK
platforms_a03i.conf OK
jobs_a03i.conf OK
expdef_a03i.conf  OK
Configuration files OK

[CRITICAL] Unhandled exception on Autosubmit: read from process failed '[Errno 5] Input/output error' : (ssh: Could not resolve hostname ithaca: Name or service not known
) ((ssh: Could not resolve hostname ithaca: Name or service not known
)) (/esnas/autosubmit/virtualenv/lib/python2.7/site-packages/saga/utils/pty_exceptions.py +40 (translate_exception)  :  e = se.BadParameter (cmsg))
Traceback (most recent call last):
  File "/esnas/autosubmit/virtualenv/lib/python2.7/site-packages/autosubmit/autosubmit.py", line 278, in parse_args
    return Autosubmit.run_experiment(args.expid)
  File "/esnas/autosubmit/virtualenv/lib/python2.7/site-packages/autosubmit/autosubmit.py", line 533, in run_experiment
    submitter.load_platforms(as_conf)
  File "/esnas/autosubmit/virtualenv/lib/python2.7/site-packages/autosubmit/platforms/submitter.py", line 110, in load_platforms
    remote_platform.service = saga.job.Service("{0}://{1}".format(adaptor, host), session=session)
  File "/esnas/autosubmit/virtualenv/lib/python2.7/site-packages/saga/job/service.py", line 115, in __init__
    url, session, ttype=_ttype)
  File "/esnas/autosubmit/virtualenv/lib/python2.7/site-packages/saga/base.py", line 101, in __init__
    self._init_task = self._adaptor.init_instance (adaptor_state, *args, **kwargs)
  File "/esnas/autosubmit/virtualenv/lib/python2.7/site-packages/saga/adaptors/cpi/decorators.py", line 57, in wrap_function
    return sync_function (self, *args, **kwargs)
  File "/esnas/autosubmit/virtualenv/lib/python2.7/site-packages/saga/adaptors/sge/sgejob.py", line 317, in init_instance
    self.shell = saga.utils.pty_shell.PTYShell(pty_url, self.session)
  File "/esnas/autosubmit/virtualenv/lib/python2.7/site-packages/saga/utils/pty_shell.py", line 242, in __init__
    posix=self.posix)
  File "/esnas/autosubmit/virtualenv/lib/python2.7/site-packages/saga/utils/pty_shell_factory.py", line 195, in initialize
    self._initialize_pty (info['pty'], info)
  File "/esnas/autosubmit/virtualenv/lib/python2.7/site-packages/saga/utils/pty_shell_factory.py", line 407, in _initialize_pty
    raise ptye.translate_exception (e)
BadParameter: read from process failed '[Errno 5] Input/output error' : (ssh: Could not resolve hostname ithaca: Name or service not known
) ((ssh: Could not resolve hostname ithaca: Name or service not known
)) (/esnas/autosubmit/virtualenv/lib/python2.7/site-packages/saga/utils/pty_exceptions.py +40 (translate_exception)  :  e = se.BadParameter (cmsg))
```

I'm not using ithaca to run my experiment.

**Comment by kinow** [comment]
In GitLab by @kserradell on Jan 8, 2016, 09:13

Using current AS version the experiment works correctly.

**Comment by kinow** [comment]
In GitLab by @jvegreg on Jan 14, 2016, 10:35

Status changed to closed by commit c97f66d8073af28ae96bc30d2c7fd617d291be7d
